atom/browser/web_dialog_helper.cc

```diff
@@ -123,7 +123,6 @@ class FileSelectHelper : public base::RefCounted<FileSelectHelper>,
       listener_.reset();
     }
     render_frame_host_ = nullptr;
-    Release();
```

Comment:
This was causing a crash because we were releasing an already-released ref.  (ref_count_ was `0`)

Reply:
It's not clear to me, is `ref_count_` _always_ 0 when we reach this point, e.g. regardless of whether or not it was cancelled?

Reply:
Based on my testing, yes, the ref count appears to be working fine now